### PR TITLE
Switch `should` for `expect` in tests - Closes #1295

### LINKED ISSUE #1295: Use should or expect consistently in tests
Child of #672 
Depends on https://github.com/LiskHQ/lisk-template/issues/75

### Expected behavior

We should use `expect` throughout our codebase. Some helpful regexes can be found here: https://github.com/LiskHQ/lisk/pull/1339#issue-289524636

### Actual behavior

We mix `should` and `expect`.

### PR DESCRIPTION
**Pull request by robladbrook**
### What was the problem?

Inconsistent use of `should` and `expect` in tests

### How did I fix it?

Mostly via:

```
find: ^(\s+)([^\s].+)\.should\.
replace: $1expect($2).to.

find: ^(\s+)should\.not\.exist\(err\);
replace: $1expect(err).to.be.null;
```

### How to test it?

Run the tests!

### Review checklist

* The PR solves #1295 
* All new code is covered with unit tests
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)

### CODE REVIEW

File: test/integration/setup/index.js [hunk 1]
```diff
@@ -45,6 +45,10 @@ module.exports = {
 				utils.logger.log('Waiting for nodes to load the blockchain');
 				network.waitForAllNodesToBeReady(configurations, cbSeries);
 			},
+			function (cbSeries) {
```
Comment (willclarktech) [inline test/integration/setup/index.js]: Is this related to this PR?

File: test/unit/api/fittings/lisk_cors.js [hunk 2]
```diff
@@ -87,9 +87,9 @@ describe('lisk_cors', function () {
 
 		cors_fititng(context, next);
 
-		next.should.have.not.been.called;
-		context.response.statusCode.should.be.equal(204);
-		should.not.exist( context.response.getHeader('my-custom-header') );
+		expect(next).to.have.not.been.called;
+		expect(context.response.statusCode).to.be.equal(204);
+		expect( context.response.getHeader('my-custom-header') ).to.be.undefined;
```
Comment (willclarktech) [inline test/unit/api/fittings/lisk_cors.js]: Did you determine these individually? Also spacing is strange.

File: test/functional/system/1.1.secondSignature.js [hunk 3]
```diff
@@ -67,9 +67,9 @@ describe('system test (type 1) - double second signature registrations', functio
 				id: transaction1.id
 			};
 			localCommon.getTransactionFromModule(library, filter, function (err, res) {
-				should.not.exist(err);
-				res.should.have.property('transactions').which.is.an('Array');
-				res.transactions.length.should.equal(0);
+				expect(err).to.be.null;
```
[the remark this answers is not in the document]
Comment (robladbrook) [reply test/functional/system/1.1.secondSignature.js]: It's not, but it is best practice: http://chaijs.com/api/bdd/#method_exist - happy to use `expect(err).to.not.exist` if preferred?

File: test/unit/api/fittings/lisk_cors.js (hunk 2 above)
Comment (robladbrook) [reply test/unit/api/fittings/lisk_cors.js]: Yes ran the test to check expected value

File: test/integration/setup/index.js (hunk 1 above)
Comment (robladbrook) [reply test/integration/setup/index.js]: Sorry, it is not!

**Comment by robladbrook** [comment]
Apologies @willclarktech - a poor PR on my part

File: test/functional/system/1.1.secondSignature.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (willclarktech) [reply test/functional/system/1.1.secondSignature.js]: No that's great, this comment came before I spotted you'd addressed these cases individually.
